crates/dpapi-cli-client/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "dpapi-cli-client"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clap = { version = "4.5", features = ["derive"] }
```

Comment:
suggestion: Since this is just a small testing tool, could you consider using xflags instead of clap?

Clap really improved on the compilation time and the binary size, but it’s still much much bigger because of the scope size is very different.

https://lib.rs/crates/xflags + https://lib.rs/crates/xflags-macros
![image](https://github.com/user-attachments/assets/22358930-3c6b-4d52-8515-b8aafecc775a) ![image](https://github.com/user-attachments/assets/dc2304e9-93f2-442f-aae1-27c1623991e4)
(no syn)

https://lib.rs/crates/clap
![image](https://github.com/user-attachments/assets/57a8008a-3fb8-4170-8abc-e7a71ba27f43)
(uses syn)

Typically, a small utility like xflags is more than enough. Feel free to use the candidate version, it doesn’t matter.

Reply:
done

Reply:
Thank you!